src/devices/bus/hp_dio/hp98544.cpp

```diff
@@ -71,6 +72,7 @@ dio16_98544_device::dio16_98544_device(const machine_config &mconfig, device_typ
 	device_dio16_card_interface(mconfig, *this),
 	m_topcat(*this, "topcat")
 {
+	m_vram.resize(0x100000);
```

Comment:
Please don't use `vector` for things that don't need dynamic resizing, and defer allocations until start where possible.